project/FilteredJacocoAgentPlugin.scala

```diff
@@ -70,17 +70,34 @@ object FilteredJacocoAgentPlugin extends AutoPlugin {
     }
   }
 
+  lazy val Jmf = config("jmf").extend(Compile)
```

Comment:
```suggestion
  lazy val jmf = config("jmf").extend(Compile)
```

Naming conventions would suggest lowercase `j`, I'd say

Reply:
Fixed in commit cf0151f8a16adaf444d05d5189b4514d7cd5b259.